selfdrive/modeld/thneed/compile.cc

```diff
@@ -0,0 +1,34 @@
+#include <string.h>
+#include "thneed.h"
+#include "../runners/snpemodel.h"
+
+#define TEMPORAL_SIZE 512
+#define DESIRE_LEN 8
+#define TRAFFIC_CONVENTION_LEN 2
+
+// TODO: This should probably use SNPE directly.
+int main(int argc, char* argv[]) {
+  #define OUTPUT_SIZE 0x10000
+  float *output = (float*)calloc(OUTPUT_SIZE, sizeof(float));
```

Comment:
do we need to add TEMPORAL_SIZE here?

Reply:
oh, we're using a separate buffer but it's part of the output buffer in driving.cc

Reply:
nah, this is a junk buffer